apps/passport-client/src/appHooks.ts

```diff
@@ -29,23 +32,22 @@ export function usePCDCollectionWithHash(): {
 }
 
 export function usePCDs(): PCD[] {
-  const { pcds } = usePCDCollectionWithHash();
+  const pcds = usePCDCollection();
```

Comment:
I *think* this change could be breaking. There was a reason I had `usePCDCollectionWithHash` here before. If we don't do it this way, there's a chance that changes to the pcd collection would not cause React to re-render. It's been a couple months since I touched the code so I could be mistaken, but I'm pretty sure that was the reason I made it be this way, at least. What was the reason you decided to change it here?

Reply:
Interesting. No particular motivation other than the fact if `  const pcds = usePCDCollection();` depends on the useEffect for it to be up to date, that feels like code smell. I wonder if that is because we mutate `state.pcds` in-place somewhere, otherwise the appSelect should trigger an udpate.
Anyways, I will revert this change since it is irrelevant.